app/services/sms_scheduler.rb

```diff
@@ -0,0 +1,9 @@
+class SMS_Scheduler
```

Comment:
I think Scheduler's function is to find all the subscription that's due today and send each one of them to the workder